DXRModules/DeusEx/Classes/DXRFixup.uc

```diff
@@ -834,3 +834,23 @@ function SetAllLampsState(bool type1, bool type2, bool type3, optional Vector lo
     }
 #endif
 }
+
+function MoveSmugglerElevator()
+{
+    local #var(prefix)DeusExMover elevator;
+    local bool street;
+
+    if (!#defined(vanilla)) return;
+
+    street = dxr.localURL == "02_NYC_STREET" || dxr.localURL == "04_NYC_STREET" || dxr.localURL == "08_NYC_STREET";
```

Comment:
As mentioned in a later comment, if you move this logic into the trigger itself, you can just pass a parameter into the class to tell it whether it's operating in a _STREET or not.

Reply:
I didn't entirely go with your suggestion here. Now, the SmugglerElevatorTracker Create function takes only a DXRando, then finds and stores the right street value. This allows for a simple `class'SmugglerElevatorTracker'.static.CreateSET(dxr)` function and all Smuggler elevator logic to be done in the class.